### DISCUSSION TITLE
Bienvenido a las discusiones de DESI High

### DISCUSSION THREAD

**Discussion by alxogm** [michaelJwilson/DESI-HighSchool#79] [General]
Usaremos este espacio para comunicarnos con otros miembros de nuestra comunidad. 

Esperamos que:

- Hagas preguntas
- Compartas ideas
- Te involucres con otros miembros de la comunidad
- Mantengas la mente abierta! Recuerda que está es una comunidad que creamos junt@s. 

Para iniciar, comenta a continuación con una pequeña introducción de ti, y que has hecho o esperas hacer como parte de esta comunidad.

**Comment by Isa-Rocha31415** [comment]
Holaaa soy Isabel Rocha y tengo 17 años, hace poco aprendí a programar y la verdad me daba un poco de miedo por la prueba final, pero como la Dr. Alma nos iba guiando en el proyecto y resolviendo dudas fue mucho más fácil hacerlo.
Los cuadernos de jupyter son muy interesantes, espero que en otro evento de DESI los veamos <3
